documentation/manual/src/main/asciidoc/en-US/modules/query.asciidoc

```diff
@@ -176,6 +196,16 @@ String query =
      "FROM StoryGame sg JOIN sg.optionalEndings ending WHERE ending.text = 'Happy ending'";
 List<String> endings = session.createQuery( query ).list();
 ----
+
+Projection of embedded id properties will work with Neo4j and MongoDB,
```

Comment:
Is it only projection or also referencing embedded id properties in WHERE clauses?